pkg/apis/apps/types.go

```diff
@@ -95,6 +95,14 @@ type RollingUpdateStatefulSetStrategy struct {
 	// Partition indicates the ordinal at which the StatefulSet should be
 	// partitioned.
 	Partition int32
+
+	// The maximum number of pods that can be unavailable during the update.
+	// Value can be an absolute number (ex: 5) or a percentage of desired pods (ex: 10%).
+	// Absolute number is calculated from percentage by rounding up. This can not be 0.
+	// Defaults to 1. This field is alpha-level and is only honored by servers that enable the
```

Comment:
is 1 the current implicit behavior?

Reply:
yes 1 is the current implicit behavior

Reply:
It looks like we are missing a change in defaults.go file

Reply:
fixed